core/translations/en.json

```diff
@@ -10,11 +10,16 @@
     "addr_mismatch__support_url": "trezor.io/support",
     "addr_mismatch__wrong_derivation_path": "Wrong derivation path for selected account.",
     "addr_mismatch__xpub_mismatch": "XPUB mismatch?",
+    "address__cancel_contact_support": "If receive address doesn't match, contact Trezor Support at trezor.io/support.",
+    "address__cancel_receive": "Cancel receive",
     "address__public_key": "Public key",
+    "address__qr_code": "QR code",
     "address__title_cosigner": "COSIGNER",
     "address__title_receive_address": "RECEIVE ADDRESS",
     "address__title_yours": "YOURS",
     "address_details__derivation_path": "Derivation path:",
+    "address_details__derivation_path_no_colon": "Derivation path",
```

Comment:
Isn't this a little bit wasteful? Can't we remove colon from `address_details__derivation_path` and append it in `tt` and `tr`. 

But I see it used in rust in `tr`, that makes it more complicated.

Reply:
Asked on slack about it, will likely change it depending on conclusion. Slight issue with appending colon is that French has a space before colon.

Reply:
Since there are only two such strings I added both variants in 3b782887634ae839ea35524831b25b97f97ce5f1 but changed the names to be consistent with `words__account_colon`.